packages/uikit-react-native/src/components/GroupChannelMessageRenderer/GroupChannelMessageReplyInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import { User } from '@sendbird/chat';
+import {
+  Avatar,
+  Box,
+  Icon,
+  PressBox,
+  Text,
+  createStyleSheet,
+  useUIKitTheme,
+} from '@sendbird/uikit-react-native-foundation';
+import { SendbirdFileMessage, SendbirdGroupChannel, SendbirdMessage, SendbirdUserMessage } from '@sendbird/uikit-utils';
+
+import { useLocalization } from '../../hooks/useContext';
+
+const AVATAR_LIMIT = 5;
+
+type Props = {
+  channel: SendbirdGroupChannel;
+  message: SendbirdMessage;
+  onPress?: (message: SendbirdUserMessage | SendbirdFileMessage) => void;
+};
+
+const createRepliedUserAvatars = (mostRepliedUsers: User[]) => {
+  if (!mostRepliedUsers || mostRepliedUsers.length === 0) return null;
+
+  const { palette } = useUIKitTheme();
+
+  return mostRepliedUsers.slice(0, AVATAR_LIMIT).map((user, index) => {
+    if (index < AVATAR_LIMIT - 1) {
+      return (
+        <Box style={styles.avatarContainer} key={index}>
+          <Avatar size={20} uri={user?.profileUrl}></Avatar>
+        </Box>
+      );
+    } else {
+      return (
+        <Box style={styles.avatarContainer} key={index}>
+          <Avatar size={20} uri={user?.profileUrl}></Avatar>
+          <Box style={styles.avatarOverlay} backgroundColor={palette.overlay01}>
+            <Icon icon={'plus'} size={14} style={styles.plusIcon} color={'white'} />
+          </Box>
+        </Box>
+      );
+    }
+  });
+};
+
+const GroupChannelMessageReplyInfo = ({ channel, message, onPress }: Props) => {
+  const { STRINGS } = useLocalization();
+  const { select, palette } = useUIKitTheme();
+
+  if (!channel || !message.threadInfo || !message.threadInfo.replyCount) return null;
+
+  const replyCountText = STRINGS.GROUP_CHANNEL_THREAD.REPLY_COUNT(message.threadInfo.replyCount || 0, 99);
+  const onPressReply = () => {
+    onPress?.(message as SendbirdUserMessage | SendbirdFileMessage);
+  };
+
+  const renderAvatars = createRepliedUserAvatars(message.threadInfo.mostRepliedUsers);
```

Comment:
```suggestion
  const avatars = createRepliedUserAvatars(message.threadInfo.mostRepliedUsers);
```

아니면 바로 `{createRepliedUserAvatars(message.threadInfo.mostRepliedUsers)}` 로 써도 좋을 것 같습니다.

Reply:
넵~ 수정하겠습니다~ 이름에 render 가 들어가는게 저번에 설명해주시기로는 컴포넌트 여부였던것 같은데? 혹시 한번더 설명 부탁드려도 될까요!

Reply:
요기 입니다!
https://github.com/sendbird/sendbird-uikit-react-native/pull/181#discussion_r1621722745

Reply:
수정 하였습니다